### PR TITLE
Range slider accent color

### PR DESCRIPTION
**Pull request by pabli24**
https://github.com/user-attachments/assets/67a30d68-125d-49a4-9f19-2a61639f5d65

### CODE REVIEW

File: src/css/global.css [hunk 1]
```diff
@@ -150,6 +150,10 @@ i {
   font-size: 16px;
 }
 
+input[type="range"] {
+  accent-color: hsl(180, 45%, 40%);
```
Comment (tophf) [inline src/css/global.css:154]: This color is already defined as `var(--accent-2)`, assuming there's no reason not to use it.

File: src/css/global-dark.css [hunk 2]
```diff
@@ -48,6 +48,9 @@
   input[type=time]::-webkit-calendar-picker-indicator {
     filter: invert(1);
   }
+  input[type="range"] {
```
Comment (tophf) [inline src/css/global-dark.css:51]: If this is Firefox-only, it should be moved to the end of the file after `/* Workarounds for FF bugs/quirks */`

File: src/css/global.css (hunk 1 above)
Comment (pabli24) [reply src/css/global.css:154]: --accent-2: hsl(180, 50%, 40%); will make the track dark in light mode for chromium browsers

Comment (tophf) [reply src/css/global.css:154]: Oh, I see, I looked at the value in the wrong file.

File: src/css/global-dark.css (hunk 2 above)
Comment (pabli24) [reply src/css/global-dark.css:51]: This is for chromium browsers to make the range track dark

**Pull request merged**